client/flutter/lib/pages/symptom_checker/symptom_checker_results_page.dart

```diff
@@ -0,0 +1,182 @@
+                  ),
+                ...result.cards
+                    .map((c) => _Card(
+                          content: c,
+                          iconColor: _severityColor(result.severity),
+                        ))
+                    .toList()
+              ],
+            ))
+        .toList();
+  }
+}
+
+class _Card extends StatelessWidget {
+  final SymptomCheckerResultCard content;
+  final Color iconColor;
+
+  _Card({@required this.content, this.iconColor});
+
+  IconData _icon(String name) {
+    if (name != null) {
+      switch (name) {
+        case 'call':
+          return FontAwesomeIcons.phoneSquareAlt;
+        case 'note':
+          return FontAwesomeIcons.notesMedical;
+        case 'home':
+          return FontAwesomeIcons.houseUser;
+        case 'bed':
+          return FontAwesomeIcons.bed;
+      }
+    }
+    return FontAwesomeIcons.commentMedical;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final TextStyle defaultTextStyle = ThemedText.htmlStyleForVariant(
+        TypographyVariant.body,
+        textScaleFactor: MediaQuery.textScaleFactorOf(context));
+    final TextStyle boldTextStyle =
+        defaultTextStyle.copyWith(fontWeight: FontWeight.w700);
+    ;
+
+    final icon = _icon(content.iconName);
+
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 8.0),
+      child: ClipRRect(
+        borderRadius: BorderRadius.circular(8.0),
+        child: Container(
+            color: CupertinoColors.white,
+            padding: EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 16.0),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Container(
+                  alignment: Alignment.center,
+                  child: FaIcon(
+                    icon,
+                    color: iconColor,
+                    size: 24 * MediaQuery.of(context).textScaleFactor,
+                  ),
+                  constraints: BoxConstraints(
+                      minWidth: 24 * MediaQuery.of(context).textScaleFactor,
+                      maxWidth: 36 * MediaQuery.of(context).textScaleFactor),
+                ),
+                SizedBox(
+                  width: 12,
+                ),
+                Flexible(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: <Widget>[
+                      Padding(
+                        // Html widget has 2 un-modifiable pixel padding, sigh...
+                        padding: EdgeInsets.symmetric(horizontal: 2),
+                        child: ThemedText(
+                          content.title,
+                          variant: TypographyVariant.body,
+                          style: TextStyle(fontWeight: FontWeight.w700),
+                        ),
+                      ),
+                      if (content.bodyHtml != null)
+                        Html(
```

Comment:
Outside of scope of this PR (unless you really want to 😜), but we use this `Html` widget with identical code so many places that it's probably worth extracting into a separate component to be more DRY

Reply:
Bug filed. https://github.com/WorldHealthOrganization/app/issues/1254